src/styles/annotator/bucket-bar.scss

```diff
@@ -11,7 +11,9 @@
     pointer-events: none;
     position: absolute;
     height: 100%;
-    width: var.$annotator-bucket-bar-width;
+    // Nov-20-2020: interim and pragmatic solution for an apparent glitch on Safari and Chrome.
```

Comment:
As an aside, if you're ever specifying a date numerically I would always recommend to use ISO dates (YYYY-MM-DD) at Hypothesis to avoid confusion between the Europeans and the Americans on the team who use different ordering in formats where the year comes last (DD-MM-YY vs MM-DD-YY).